samples/tfm/provisioning_image/prj.conf

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2022 Nordic Semiconductor ASA
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
why is the copyright year changed ?
I assume this file is still copyrighted from 2022.